src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -129,5 +193,6 @@ void PromptUserForLicenseIfTrialHasExpired()
         License license;
 
         static ILog Logger = LogManager.GetLogger(typeof(LicenseManager));
+        static HttpClient httpClient = new HttpClient();
```

Comment:
Here's the reasoning:
http://bartwullems.blogspot.com.co/2016/09/dont-dispose-your-httpclient.html